controllers/articles.controller.js

```diff
@@ -8,4 +8,11 @@ function getArticlesById(req, res, next){
     .catch(next)
 }
 
-module.exports = { getArticlesById }
\ No newline at end of file
+function getArticles(req, res, next){
+    return fetchArticles().then((articles) => {
+        res.status(200).send({ articles })
+    })
+    .catch.next
```

Comment:
what is this doing?

Reply:
A mistake on my half, I mistyped that last bit! I've changed it and will push it through with the next pull request! :D